README.md

```diff
@@ -9,8 +9,10 @@ Idea from Nick Carter, initial implementation from Richard Smith.
     Converts one (or several) .ninja_log files into chrome's about:tracing format
 
     Usage:
-        rm $BUILDDIR/.ninja_log && ninja -C $BUILDDIR
+        ninja -C $BUILDDIR
         ninjatracing $BUILDDIR/.ninja_log > trace.json
 
-    (If you don't have time for a clean build, at least run
-    `ninja -C $BUILDDIR -t recompact` first.)
+    By default this will show build timing results for the most recent (possibly
+    incremental) build. To show build timing results for every target, whether
+    built in the last build or previously, use --showall. Note that this will
+    overlap multiple builds and will thus exaggerate build parallelism.
```

Comment:
I definitely can, and I'll happily defer to your judgment on this. But...

I'm not sure what value recompact actually adds. My understanding is that it throws out the old entries for all of the build targets. However this has never made any difference to the results of ninjatracing because ninjatracing always overwrites the old entries. I think that's the case anyway.

That is, my understanding is that -t recompact is a handy way of shrinking the size of your .ninja_log file but does not make any difference to your builds or to ninjatracing.